plugin.php

```diff
@@ -367,6 +367,7 @@ function create_initial_json_routes() {
 			'methods'   => WP_JSON_Server::READABLE,
 			'callback'  => array( $controller, 'get_items' ),
 			'permission_callback' => array( $controller, 'get_items_permissions_check' ),
+			'schema'    => $controller->get_item_schema(),
```

Comment:
This really sucks. It would be better if the server just knew which named resource it was dealing with, and how to find that schema.

Related #945, #577